testing/scripts/test_helpdesk_pw.py

```
@@ -4570,46 +4569,47 @@ def test_72_portal_v2_renders_conversation(page):
 
     page.goto(portal_url)
     try:
-        page.locator(".swh-portal-v2").wait_for(timeout=10000)
+        portal_root(page).wait_for(timeout=10000)
     except Exception:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)